### PR TITLE
Restructure project directories & add contrib app

### PR DESCRIPTION
**Pull request by whmii**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    root: "_contrib/source",
+    port: 8000,
+    livereload: true
+  })
+);
+
+gulp.task('build', [ 'sass', 'sasslint', 'javascripts', 'eslint' ]);
+
+gulp.task('default', [ 'build', 'connect' ], () =>
+  gulp.watch( [paths.scss, paths.js] , [ 'sass', 'sasslint', 'javascripts', 'eslint' ])
```
Comment (enatario) [inline gulpfile.js:66]: Do you need `sasslint` and `eslint` in the watch task? I'm wondering if you can nix those unless you're often changing them out. I've also found that gulp tasks can fail (depending on your linter settings) if you're switching things around while it's watching.

File: _contrib/source/index.html [hunk 2]
```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Have fun ❤︎</title>
+    <link rel="stylesheet" href="assets/stylesheets/application.css">
+    <script type="text/javascript" src="assets/javascripts/application.js"></script>
```
Comment (enatario) [inline _contrib/source/index.html:9]: Maybe throw the js at the bottom of `body` so it's not render blocking

File: _contrib/source/index.html [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+  </head>
+  <body class="container">
+    <br>
```
Comment (enatario) [inline _contrib/source/index.html:12]: Could you use a margin top in lieu of `br`s?

File: _contrib/source/index.html [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+    <p class="card">
+      Lorem ipsum dolor sit amet, <a href="#" class="text-link">consectetur adipisicing</a> elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+      quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+      consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+      cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+      proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+
+    <h2>
```
Comment (enatario) [inline _contrib/source/index.html:28]: Might be nice to wrap each portion that begins with an `h2` in its own `section` for clear division of content. Or something like all things with `button` in a `section` and all `checkbox` things in a `section`

File: _contrib/source/index.html [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+    </h2>
+    <form>
+      <input type="button" value="button" class="form-input">
+    </form>
+
+    <h2>
+      `input type="form-checkbox"`
+    </h2>
+    <form>
+      <div class="form-inline-group">
```
Comment (enatario) [inline _contrib/source/index.html:52]: Ooo this might be a good opportunity to use `fieldset` in lieu of `div`!

File: _contrib/source/index.html [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+      </div>
+    </form>
+
+    <h2>
+      `input type="image"`
+    </h2>
+    <form>
+      <input
+        type="image"
+        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/010.png"
```
Comment (enatario) [inline _contrib/source/index.html:100]: 😂

**Review by enatario — COMMENTED** [review]
This is awesome!!! Just had a few suggestions :)

File: _contrib/source/index.html (hunk 3 above)
Comment (whmii) [reply _contrib/source/index.html:12]: Typically I would but this is not meant to be functioning html and only exists as an example page that one can develop against. As such it doesn't have any css for itself, just that of the library.

**Pull request merged**